addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -114,6 +114,9 @@
                                 <field name="next_departure" class="oe_inline"/>.
                             </strong>
                         </div>
+                        <div>
```

Comment:
Doesn't it introduce some "margin" due to div when warning message is void ?

Reply:
No :) 

![screenshot_spect](https://user-images.githubusercontent.com/54316226/121666275-5b997700-caa9-11eb-9e7c-eb9ce1ddeb37.png)